latest/index.bs

```diff
@@ -233,34 +234,897 @@ keys as specified below for discovering certain types of data, especially images
+in the container, its axes have `"type":"array"`, are unitless, and have default "name"s. The ith axis has `"name":"dim_i"`
+(these are the same default names used by [xarray](https://docs.xarray.dev/en/stable/user-guide/terminology.html)).
+<div class=example>
+For example, a 3D array at path `/my/data/array` defines the coordinate system:
+
+```json
+{
+    "name" : "/my/data/array",
+    "axes" : [
+        {"name": "dim_0", "type": "array"},
+        {"name": "dim_1", "type": "array"},
+        {"name": "dim_2", "type": "array"}
+    ]
+}
+```
+
+though this object should not and need not explicitly appear in metadata. 
+</div>
+
+
+The dimensionality of each array coordinate system equals the dimensionality of its corresponding zarr array.  The axis with
+name `"dim_i"` is the ith element of the `"axes"` list. The axes and their order align with the `shape`
+attribute in the zarr array attributes (in `.zarray`), and whose data depends on the byte order used to store
+chunks. As described in the [zarr array metadata](https://zarr.readthedocs.io/en/stable/spec/v2.html#arrays),
+the last dimension of an array in "C" order (row-major) are stored contiguously. For an array in "F"
+(column-major) order, the elements of the first dimension are stored contiguously.
+
+<div class=example>
+For example, if `/my/data/array/.zarray` contains:
+
+```json
+{
+    "chunks": [ 4, 3, 5 ],
+    "compressor": null,
+    "dtype": "|u1",
+    "fill_value": 0,
+    "filters": null,
+    "order": "C",
+    "shape": [ 4, 3, 5 ],
+    "zarr_format": 2
+}
+```
+
+Then `dim_0` has length 4, `dim_1` has length 3, and `dim_2` has length 5.
+</div>
+
+The name and axes names MAY be customized by including a `arrayCoordinateSystem` field in
+the `.zattr` metadata of the array whose value is a coordinate system object. The length of
+`axes` MUST be equal to the dimensionality. The value of `"type"` for each object in the 
+axes array MUST equal `"array"`.
+
+<div class=example>
+
+<pre class=include-code>
+path: examples/coordSystems/arrayCoordSys.json
+highlight: json
+</pre>
+
+</div>
+
+
+### Coordinate convention
+
+**The pixel/voxel center is the origin of the continuous coordinate system.**
+
+It is vital to consistently define relationship between the discrete/arrray and continuous/interpolated
+coordinate systems. A pixel/voxel is the continuous region (rectangle) that corresponds to a single sample
+in the discrete array, i.e., the area corresponding to nearest-neighbor (NN) interpolation of that sample.
+The center of a 2d pixel corresponding to the origin `(0,0)` in the discrete array is the origin of the continuous coordinate
+system `(0.0, 0.0)` (when the transformation is the identity). The continuous rectangle of the pixel is given by the
+half-open interval `[-0.5, 0.5) x [-0.5, 0.5)`. See chapter 4 and figure 4.1 of the ITK Software Guide [[itk]].
```

Comment:
Mixture of brackets here: `[-0.5, 0.5) x [-0.5, 0.5)`?

Reply:
Intended that way. Meant to mean -0.5 is included, +0.5 is excluded. Like the [left closed interval here](https://en.wikipedia.org/wiki/Interval_(mathematics)#Classification_of_intervals).
Do you have a preferred way of writing that? I can be more explicit.

Reply:
I'm not familiar with that terminology, but then I'm probably not the intended audience here and those that are will likely understand OK. I guess you could just add "...(-0.5 is included, +0.5 is excluded)".

Reply:
A wide audience is best, so I should have been more explicit.  I like your suggestion 

> (-0.5 is included, +0.5 is excluded)

and so added it just now https://github.com/ome/ngff/pull/138/commits/0df402d185120ce166ac5c318c35039b8a999fe2